FWCore/ParameterSet/python/Mixins.py

```diff
@@ -422,6 +422,8 @@ def clone(self, *args, **params):
             args.append(None)
         
         _modifyParametersFromDict(myparams, params, self._Parameterizable__raiseBadSetAttr)
+        if self._Parameterizable__validator is not None:
+            myparams["allowAnyLabel_"] = self._Parameterizable__validator
```

Comment:
What if the `clone()` has `clone(allowAnyLabel_ = cms...)`or `clone(allowAnyLabel_ = None)`?

Reply:
What behavior would we want in that case? It seems kind of suspect to me :).